geode-connectors/src/main/java/org/apache/geode/connectors/jdbc/internal/cli/CreateMappingFunction.java

```diff
@@ -46,11 +49,21 @@ public CliFunctionResult executeFunction(FunctionContext<ConnectorService.Region
     return new CliFunctionResult(member, true, message);
   }
 
+  private void verifyRegionExists(FunctionContext<RegionMapping> context,
```

Comment:
👍

Reply:
Did you see something that needed to be changed on "verifyRegionExists"?